File: time_logic.cpp
#include <iostream>
using namespace std;

int main() {
    int totalSeconds;

    cout << "Enter the number of seconds: ";
    cin >> totalSeconds;

    int totalHour = totalSeconds / 3600;
    int remainSecond = totalSeconds % 3600;

    int totalMinute = remainSecond / 60;
    int second = remainSecond % 60;

    cout << totalSeconds << " seconds is "
         << totalHour << " hours, "
         << totalMinute << " minutes, and "
         << second << " seconds." << endl;

    return 0;
}

File: time_logic.py
totalseconds=int(input("Enter the number of seconds "))

totalhour = totalseconds // 3600

remainsecond = totalseconds % 3600



totalminute = remainsecond // 60

second = remainsecond % 60

print(totalseconds , "second is " , totalhour , "hours , " , totalminute , "minute , and " , second , "seconds")

